addons/popochiu/editor/config/config.gd

```diff
@@ -114,6 +118,11 @@ static func is_use_translations() -> bool:
 static func get_dialog_style() -> int:
 	return _get_project_setting(DIALOG_STYLE, DialogStyle.ABOVE_CHARACTER)
 
+static func should_talk_gibberish() -> bool:
```

Comment:
I like the naming of these functions, but they are inconsistent with the siblings in the Config class.

We can introduce this exception but in this case, I'd consider propagating this semantics to the rest of the plugin/engine over time, so we don't have mix-and-matches, or all contributions writing functions at their taste.

@mapedorr what do you think?

Reply:
about that function - I did follow what was in beta1 :
```
func should_remove_source_files() -> bool:
	return _get_editor_setting(_REMOVE_SOURCE_FILES_KEY, true)
```

But I can change that :)

Reply:
Hi @mefjak ! No need to rename it, I am dumb! Ignore my comment, and let's keep this. I also like it, as I wrote :D

Sorry

Reply:
No problem mate :)

Reply:
Yes. I like (a lot) the use of the `should_` prefix for some boolean types. If sometimes I don't use it, it's because I'm afraid people will laugh at me when they read my code.